addons/mail/static/src/scss/activity_view.scss

```diff
@@ -69,10 +69,13 @@
         padding: 8px 8px;
         cursor: pointer;
 
-        > img {
-            width: 32px;
-            max-height: 32px;
-            margin-right: 16px;
```

Comment:
Are you sure you're not breaking the design in some places ?

Reply:
@msh-odoo And about this comment ? If any activity view record has an image (other than the avatar one), you may impact it.